server/src/server.ts

```diff
@@ -63,7 +63,7 @@ import { dbConnect } from "./db";
 	var aclOptions = {
 		roles: config.acl.roles,
 		defaultRoles: ["guest"],
-		userRoles: ["user"],
+		// userRoles: ["user"],

```

Comment:
Je nejaky duvod, proc je to jenom zakomentovane a ne smazane?

Reply:
Neni...